test/CMakeLists.txt

```diff
@@ -135,3 +135,9 @@ add_umf_test(NAME base_alloc
 add_umf_test(NAME base_alloc_linear
             SRCS ${BA_SOURCES_FOR_TEST} test_base_alloc_linear.cpp
             LIBS umf_utils)
+if(LINUX)
+    # TODO: fix this on windows
+    add_umf_test(NAME base_alloc_global
+                SRCS ${BA_SOURCES_FOR_TEST} pools/pool_base_alloc.cpp malloc_compliance_tests.cpp
```

Comment:
hmm... why is here (and not only here) `malloc_compliance_tests.cpp`...? Is there some magic I don't understand or just a copy-paste issue...? 😉

Reply:
All is OK here. The `base_alloc_global` test run the test cases defined in `malloc_compliance_tests.cpp`.

Reply:
@lukaszstolarczuk a bit more detailed explanation ...
There are 4 functions-tests defined in `malloc_compliance_tests.cpp`:
```c
void malloc_compliance_test(umf_memory_pool_handle_t hPool);
void calloc_compliance_test(umf_memory_pool_handle_t hPool);
void realloc_compliance_test(umf_memory_pool_handle_t hPool);
void free_compliance_test(umf_memory_pool_handle_t hPool);
```
Those 4 functions are used in 4 gtest `TEST_P()` tests in the `poolFixtures.hpp` header:
```c
TEST_P(umfPoolTest, malloc_compliance)
TEST_P(umfPoolTest, calloc_compliance) 
TEST_P(umfPoolTest, realloc_compliance)
TEST_P(umfPoolTest, free_compliance)
```
... that is included by `pool_base_alloc.cpp` and other tests ...